### ISSUE TITLE
After finishing downloading video you must force to log-in again to download another one.

### ISSUE THREAD

**Issue by amodia** [learningequality/ka-lite#4204]
**Branch:** `0.14.x`
**Expected behavior:** Admin user must not log-in again after finish downloading video.
**Current behavior:** Admin user force to log-in again to download another video.

**Steps to reproduce:**
 1. Log in as admin.
 2. Navigate to video tab.
 3. Download any video and wait to finish.
 4. Try to download video again and you need to log-in to download another video.

This issue may related to #4181 

![ka-lite-a](https://cloud.githubusercontent.com/assets/8663934/9081058/ca8bb140-3b8b-11e5-81bd-1b4a0c0757f9.gif)

**Comment by EdDixon** [comment]
I have also been replicating this issue but am being immediately logged off when clicking on the video tab on MAC and Linux 0.14 installations. I do feel it maybe related to #4118 but also on the linux version last tested here, it seemed to be logging me off at random as well. I will try to test a more recent Linux version for more clues or if I can find a means to reliably replicate that.

**Comment by aronasorman** [comment]
Happens too for language packs. However I don't think this should be a release blocker.

**Comment by EdDixon** [comment]
@benjaoming on Debian with version a12 exhibiting same behavior, immediately requiring the admin user to login again with the following security warning in console: 
```
Use of getPreventDefault() is deprecated. Use defaultPrevented instead. 
Password fields present on an insecure (http://) page. This is a security risk that allows user login credentials to be stolen.
```
@aron ? escalate to blocker ? sounds important enough? but also easy enough to handle?

**Comment by aronasorman** [comment]
I still feel that it's not a blocker, though it certainly deserves to be fixed.

**Comment by EdDixon** [comment]
Awesome!

**Comment by aronasorman** [comment]
Fixed in https://github.com/learningequality/ka-lite/commit/9961909eec50a9cbae64654b0bf9c9a53b3341ee. @radinamatic @amodia or @EdDixon can one of you guys test it out and see if it's fixed on your side?

**Comment by radinamatic** [comment]
Does seem to be fixed, @aronasorman ! :+1: 

I updated Spanish pack on my 0.14.x branch and I'm downloading Lao (curious to see that one, beautiful letters ;) and it did not log me out.

Anyway, I believe you pushed that fix after @MCGallaspy compiled the Windows installer, so make sure it gets included in the final release!

**Comment by aronasorman** [comment]
Nice!

Just a word of warning, [there are known issues with a combination of SECRET_KEY being public and signed cookies](https://docs.djangoproject.com/en/1.5/topics/http/sessions/#using-cookie-based-sessions.) , but they're to be addressed in this issue: #4213 

Thanks for testing @radinamatic! Closing.
